source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/DomainTests/ChargeIngestionTests.cs

```diff
@@ -134,8 +144,10 @@ public async Task When_ChargeInformationIsReceived_Then_ChargeCreatedIntegration
             public async Task When_ChargePricesAreReceived_Then_ChargePricesUpdatedIntegrationEventsArePublished()
             {
                 // Arrange
-                var (request, correlationId) = await _authenticatedHttpRequestGenerator
-                    .CreateAuthenticatedHttpPostRequestAsync(EndpointUrl, ChargeDocument.TariffPriceSeries);
+                var (request, correlationId) =
+                    _gridAccessProviderAuthenticatedHttpRequestGenerator.CreateAuthenticatedHttpPostRequest(
+                        EndpointUrl,
+                        ChargeDocument.TariffPriceSeries);
```

Comment:
```suggestion
                        EndpointUrl, ChargeDocument.TariffPriceSeries);
```
and likewise downwards

Reply:
Fixed in https://github.com/Energinet-DataHub/geh-charges/pull/1533/commits/64882bd8b2daeb000447b232abdda48928b9bd70